src/messages.lisp

```diff
@@ -159,6 +159,11 @@
      (|seed|
       :documentation "PRNG seed. Set this to guarantee repeatable results."
       :type :integer
+      :required nil)
+
+     (|interleaver|
+      :documentation "Fixed Clifford to interleave through an RB sequence."
```

Comment:
should we make a note that this Clifford is expressed as a Quil program, as was done in the `gateset` doc?